### DISCUSSION TITLE
Password Reset Link

### DISCUSSION THREAD

**Discussion by Anomalia2022** [modoboa/modoboa#3006] [Help]
My community and I are trying to set up the system to reset user’s emails via the link on the login page. Upon entering their email and pressing send, they don’t receive an email. I do have all their emails forwarded to their gmail as well but neither the internal nor external emails received any mail. And I also configured the notifications noreply email.

**Comment by Spitfireap** [comment]
Hi, is it from legacy interface or new-admin ?

**Reply by Anomalia2022** [reply]
On the new admin panel
